.github/CODEOWNERS

```diff
@@ -1,13 +1,13 @@
 #cpp code owners
-cpp/               @rapidsai/<repo>-cpp-codeowners
+cpp/               @rapidsai/cucim-cpp-codeowners
```

Comment:
Would it be possible to add @gigony and myself to this team?

Reply:
done: https://github.com/orgs/rapidsai/teams/cucim-cpp-codeowners